packages/webapp/src/delegates/api/fixtures.ts

```diff
@@ -0,0 +1,115 @@
+import { MemberData, MemberStats } from "members";
+
+export const fixtureMemberStats: MemberStats = {
+    active_members: 4,
+    pending_members: 0,
+    completed_waiting_inductions: 0,
+    ranks: [4, 1],
+};
+
+export const fixtureMembers: MemberData[] = [
+    {
+        // a Chief Delegate
+        templateId: 84431,
+        createdAt: 1621635292500,
+        name: "Eden Member 12",
+        image: "QmSbh7viN2Xd9g7FnccLsfhBcNsKBf7Z1Q7ZqmsDjLnvQf",
+        account: "edenmember12",
+        bio: "This is a test.",
+        attributions: "https://rivkahfineart.com",
+        inductionVideo: "QmTYqoPYf7DiVebTnvwwFdTgsYXg2RnuPrt8uddjfW2kHS",
+        socialHandles: {
+            eosCommunity: "brandonfancher",
+            twitter: "brandonfancher",
+            linkedin: "test",
+            telegram: "brandonfancher",
+            blog: "brandonfancher.com",
+        },
+        status: 1, //active
+        election_participation_status: 0, // no_donation is kinda the default if other special cases don't apply
+        election_rank: 1,
+        representative: "edenmember11", // "parent" field
+    },
+    {
+        templateId: 84432,
+        createdAt: 1621635329000,
+        name: "Eden Member 11",
```

Comment:
So in this case, `Eden Member 11` is the Head Chief, right?

Reply:
right